guides/bug_report_templates/action_view_gem.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+
+gemfile(true) do
+  source "https://rubygems.org"
+
+  git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+  gem "actionpack", "~> 7.1.0"
+  gem "actionview", "~> 7.1.0"
+end
+
+require "minitest/autorun"
+require "action_view"
+
+class BuggyViewTest < ActionView::TestCase
+  module Helpers
+    def upcase(value)
+      value.upcase
+    end
+  end
+
+  def test_stuff
+    view.extend Helpers
+
+    render inline: <<~ERB, locals: { key: "value" }
+      <p><%= upcase(key) %></p>
+    ERB
+
+    assert_pattern { rendered.html.at("p") => { content: "VALUE" } }
```

Comment:
Pattern  matching doesn't work on 2.7, which is still supported by Rails 7.1.
```suggestion
    # pattern matching requires at least Ruby 3.0.
    assert_pattern { rendered.html.at("p") => { content: "VALUE" } }
```